.github/workflows/release_portable_linux_packages.yml

```diff
@@ -221,6 +216,26 @@ jobs:
           echo "------------"
           du -h -d 1 ${{ env.OUTPUT_DIR }}/build/dist/rocm
 
+      # TODO(#1522): upload to therock-${{ env.RELEASE_TYPE }}-artifacts instead of therock-artifacts
+      #   therock-dev-artifacts
+      #   therock-nightly-artifacts
+      #   therock-prerelease-artifacts
+      - name: Configure AWS Credentials for non-forked repos
+        if: ${{ always() && !github.event.pull_request.head.repo.fork }}
```

Comment:
I copied these conditions from other workflows and I'm not sure about them here. This workflow should not be running on pull requests. Do we want to _always_ upload or skip if cancelled? Logs are useful for debugging, but we don't want to publish partial artifacts I think?

Reply:
Maybe we want the same setup as for pytorch wheels?

* Always upload logs if we have them
* Upload artifacts to staging before we run tests
* Copy artifacts out of staging after tests pass

I'm less concerned now about pushing nightly releases that fail tests given that we have stable releases running. Is making this more complex worth it?

Releases should only be triggered from commits that passed builds/tests anyways (though that isn't quite true today), so I think the risks of uploading broken artifacts here are low.